diff --git a/libmamba/CMakeLists.txt b/libmamba/CMakeLists.txt
index c7f1d0f6..fc66ac4e 100644
--- a/libmamba/CMakeLists.txt
+++ b/libmamba/CMakeLists.txt
@@ -509,6 +509,7 @@ macro(libmamba_create_target target_name linkage output_name)
             set(
                 REQUIRED_STATIC_DEPS
                 libcurl.a
+                libpsl.a
                 libssh2.a
                 libgssapi_krb5.a
                 libkrb5.a
@@ -622,6 +623,7 @@ macro(libmamba_create_target target_name linkage output_name)
                 libssl_static.lib
                 libcrypto_static.lib
                 libcurl_a.lib
+                libpsl.a
             )
 
             set(STATIC_DEPS "")
