From c7f077338f4cc9edde5e7e858725ed91a84fe746 Mon Sep 17 00:00:00 2001
From: AntoinePrv <AntoinePrv@users.noreply.github.com>
Date: Mon, 4 Sep 2023 16:27:13 +0200
Subject: [PATCH] Change Windows static lib name

---
 libmamba/CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/libmamba/CMakeLists.txt b/libmamba/CMakeLists.txt
index 5aa0e4cf..87c250e3 100644
--- a/libmamba/CMakeLists.txt
+++ b/libmamba/CMakeLists.txt
@@ -442,8 +442,8 @@ macro(libmamba_create_target target_name linkage output_name)
             find_library(BZIP2_LIBRARIES NAMES bz2)
             find_library(CRYPTO_LIBRARIES NAMES libcrypto)
 
-            find_library(LIBSOLV_BUILD_STATICRARIES NAMES solv)
-            find_library(LIBSOLVEXT_BUILD_STATICRARIES NAMES solvext)
+            find_library(LIBSOLV_BUILD_STATICRARIES NAMES solv_static)
+            find_library(LIBSOLVEXT_BUILD_STATICRARIES NAMES solvext_static)
             find_package(reproc++ CONFIG REQUIRED)
             find_package(tl-expected REQUIRED)
 
-- 
2.41.0

