diff --git a/libmamba/CMakeLists.txt b/libmamba/CMakeLists.txt
index 07387316..ad2f3194 100644
--- a/libmamba/CMakeLists.txt
+++ b/libmamba/CMakeLists.txt
@@ -421,7 +421,7 @@ macro(libmamba_create_target target_name linkage output_name)
 
             if (APPLE)
                 set (MAMBA_FORCE_DYNAMIC_LIBS resolv c++abi)
-                target_link_options(${target_name} PRIVATE -static-libstdc++)
+                target_link_options(${target_name} PRIVATE -nostdlib++)
             elseif (UNIX)
                 set(MAMBA_FORCE_DYNAMIC_LIBS rt dl resolv)
                 target_link_options(${target_name} PUBLIC -static-libstdc++ -static-libgcc)
diff --git a/micromamba/CMakeLists.txt b/micromamba/CMakeLists.txt
index 8264b795..cbef37bf 100644
--- a/micromamba/CMakeLists.txt
+++ b/micromamba/CMakeLists.txt
@@ -77,6 +77,9 @@ message(STATUS "Micromamba linkage: ${MICROMAMBA_LINKAGE}")
 
 if (${MICROMAMBA_LINKAGE} STREQUAL "STATIC")
     target_link_libraries(micromamba PRIVATE libmamba-static)
+    if(APPLE)
+        target_link_options(micromamba PRIVATE -nostdlib++)
+    endif()
     if (WIN32)
         set_target_properties(micromamba PROPERTIES MSVC_RUNTIME_LIBRARY "MultiThreaded$<$<CONFIG:Debug>:Debug>")
     endif ()
