diff --git a/libmamba/CMakeLists.txt b/libmamba/CMakeLists.txt
index 06857a4..dd7cc6a 100644
--- a/libmamba/CMakeLists.txt
+++ b/libmamba/CMakeLists.txt
@@ -280,7 +280,7 @@ macro(libmamba_create_target target_name linkage deps_linkage output_name)
             find_library(CHARSET_LIBRARY NAMES libcharset)
             message("Found: ${LIBXML2_LIBRARY} ${ICONV_LIBRARY} ${CHARSET_LIBRARY}")

-            target_link_libraries(${target_name} PRIVATE
+            target_link_libraries(${target_name} PUBLIC
                 ${CRYPTO_LIBRARIES}
                 ${LibArchive_LIBRARY}
                 ${LIBXML2_LIBRARY}
diff --git a/micromamba/CMakeLists.txt b/micromamba/CMakeLists.txt
index ef0558d..5490d84 100644
--- a/micromamba/CMakeLists.txt
+++ b/micromamba/CMakeLists.txt
@@ -59,6 +59,9 @@ message(STATUS "Micromamba linkage: ${MICROMAMBA_LINKAGE}")

 if (${MICROMAMBA_LINKAGE} STREQUAL "FULL_STATIC")
     target_link_libraries(micromamba PRIVATE libmamba-full-static)
+    if (WIN32)
+        set(CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS_RELEASE} /MT")
+    endif ()
 elseif (${MICROMAMBA_LINKAGE} STREQUAL "STATIC")
     target_link_libraries(micromamba PRIVATE libmamba-static)
 else ()
