diff --git a/libmamba/CMakeLists.txt b/libmamba/CMakeLists.txt
index 06857a4..dd7cc6a 100644
--- a/libmamba/CMakeLists.txt
+++ b/libmamba/CMakeLists.txt
@@ -280,7 +280,7 @@ macro(libmamba_create_target target_name linkage deps_linkage output_name)
             find_library(CHARSET_LIBRARY NAMES libcharset)
             message("Found: ${LIBXML2_LIBRARY} ${ICONV_LIBRARY} ${CHARSET_LIBRARY}")

-            target_link_libraries(${target_name} PRIVATE
+            target_link_libraries(${target_name} PUBLIC
                 ${CRYPTO_LIBRARIES}
                 ${LibArchive_LIBRARY}
                 ${LIBXML2_LIBRARY}
diff --git a/micromamba/CMakeLists.txt b/micromamba/CMakeLists.txt
index ef0558d..5490d84 100644
--- a/micromamba/CMakeLists.txt
+++ b/micromamba/CMakeLists.txt
@@ -59,6 +59,9 @@ message(STATUS "Micromamba linkage: ${MICROMAMBA_LINKAGE}")

 if (${MICROMAMBA_LINKAGE} STREQUAL "FULL_STATIC")
     target_link_libraries(micromamba PRIVATE libmamba-full-static)
+    if (WIN32)
+        set(CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS_RELEASE} /MT")
+    endif ()
 elseif (${MICROMAMBA_LINKAGE} STREQUAL "STATIC")
     target_link_libraries(micromamba PRIVATE libmamba-static)
 else ()
diff --git a/libmamba/CMakeLists.txt b/libmamba/CMakeLists.txt
index 5f9ac91..dbc5bcd 100644
--- a/libmamba/CMakeLists.txt
+++ b/libmamba/CMakeLists.txt
@@ -434,49 +434,3 @@ endif()
 
 # Installation
 # ============
-
-include(GNUInstallDirs)
-include(CMakePackageConfigHelpers)
-
-set(LIBMAMBA_CMAKECONFIG_INSTALL_DIR "${CMAKE_INSTALL_LIBDIR}/cmake/${PROJECT_NAME}" CACHE STRING "install path for libmambaConfig.cmake")
-
-install(TARGETS ${libmamba_targets}
-        EXPORT ${PROJECT_NAME}-targets
-        ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
-        LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
-        RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR})
-
-install(DIRECTORY "${LIBMAMBA_INCLUDE_DIR}/"
-        DESTINATION "${CMAKE_INSTALL_INCLUDEDIR}"
-        FILES_MATCHING
-        PATTERN "*.hpp"
-        PATTERN "*.h")
-
-# Makes the project importable from the build directory
-export(EXPORT ${PROJECT_NAME}-targets
-        FILE "${CMAKE_CURRENT_BINARY_DIR}/${PROJECT_NAME}Targets.cmake")
-
-# Configure 'mambaConfig.cmake' for a build tree
-set(MAMBA_CONFIG_CODE "####### Expanded from \@MAMBA_CONFIG_CODE\@ #######\n")
-set(MAMBA_CONFIG_CODE "${MAMBA_CONFIG_CODE}set(CMAKE_MODULE_PATH \"${CMAKE_CURRENT_SOURCE_DIR}/cmake;\${CMAKE_MODULE_PATH}\")\n")
-set(MAMBA_CONFIG_CODE "${MAMBA_CONFIG_CODE}##################################################")
-configure_package_config_file(${PROJECT_NAME}Config.cmake.in
-                                "${CMAKE_CURRENT_BINARY_DIR}/${PROJECT_NAME}Config.cmake"
-                                INSTALL_DESTINATION ${PROJECT_BINARY_DIR})
-
-# Configure 'mambaConfig.cmake' for an install tree
-set(MAMBA_CONFIG_CODE "")
-configure_package_config_file(${PROJECT_NAME}Config.cmake.in
-                                "${CMAKE_CURRENT_BINARY_DIR}/CMakeFiles/${PROJECT_NAME}Config.cmake"
-                                INSTALL_DESTINATION ${LIBMAMBA_CMAKECONFIG_INSTALL_DIR})
-
-
-write_basic_package_version_file(${CMAKE_CURRENT_BINARY_DIR}/${PROJECT_NAME}ConfigVersion.cmake
-                                    VERSION ${MAMBA_VERSION_MAJOR}.${MAMBA_VERSION_MINOR}.${MAMBA_VERSION_PATCH}
-                                    COMPATIBILITY AnyNewerVersion)
-install(FILES ${CMAKE_CURRENT_BINARY_DIR}/CMakeFiles/${PROJECT_NAME}Config.cmake
-                ${CMAKE_CURRENT_BINARY_DIR}/${PROJECT_NAME}ConfigVersion.cmake
-                DESTINATION ${LIBMAMBA_CMAKECONFIG_INSTALL_DIR})
-install(EXPORT ${PROJECT_NAME}-targets
-        FILE ${PROJECT_NAME}Targets.cmake
-        DESTINATION ${LIBMAMBA_CMAKECONFIG_INSTALL_DIR})
